exercises/sum.cpp

```diff
@@ -6,3 +6,19 @@
   Insert the second number: 2
   Sum: 3
 */
+
+
+#include <iostream>
+
+using namespace std;
+
+int main() {
+    int num1, num2;
+    cout << "Insert the first number: ";
+    cin >> num1;
+    cout << "Insert the second number: ";
+    cin >> num2;
+    int sum = num1 + num2;
+    cout << "Sum: " << sum << endl;
+    return 0;
```

Comment:
Il `return 0` è poco esplicativo di per sè.
Potresti fare un:
`#define EXIT_SUCCESS 0` e inserire alla fine del main `return EXIT_SUCCESS;` in modo tale da incrementare la leggibilità del codice.

```suggestion
    return EXIT_SUCCESS;
```